packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -592,6 +600,13 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
             readonlyRows,
             lockedRows,
         } = this.props;
+
+        // initial implementation of drag handle fill actions only support single column selection
+        const sortedSelectionCellKeys = !editorModel.hasMultipleColumnSelection()
```

Comment:
Consider making `lastSelection()` a method on `EditorModel`.

```ts
lastSelection(colIdx: number, rowIdx: number): boolean {
    let cellKeys = [];

    // Initial implementation of drag handle fill actions only support single column selection
    if (!this.hasMultipleColumnSelection()) {
        if (this.hasMultipleSelection()) {
            cellKeys = this.getSortedSelectionKeys();
        } else {
            cellKeys = [this.getSelectionKey()];
        }
    }

    if (cellKeys.length === 0) {
        return false;
    }

    return cellKeys.indexOf(genCellKey(colIdx, rowIdx)) === cellKeys.length - 1;
}
```

and then it can be removed from being an input to `inputCellFactory` resulting in a line that looks like:

```tsx
lastSelection={editorModel ? editorModel.lastSelection(colIdx, rn) : false}
```

Reply:
Sounds good. Done (thanks for the patch)